kong/templates/nginx_kong.lua

```diff
@@ -58,6 +58,9 @@ log_format kong_log_format '$remote_addr - $remote_user [$time_local] '
 
 # Load variable indexes
 lua_kong_load_var_index default;
+lua_kong_load_var_index $http_x_kong_request_debug;
```

Comment:
If we do need these variables we could index them in lua-kong-module by default.